### PR TITLE
Adding statsd_exporter package

### PR DESCRIPTION
**Pull request by lzap**
This is an attempt to provide package for end-to-end telemetry
integration with prometheus or pcp. I am working on a document how to
set things up, but to be able to effectively export telemetry data from
Rails, the most reasonable approach today is to use statsd and
node_exporter to get to metrics in Prometheus format.

This is a simple SPEC which builds from sources, the upstream project
has vendor/ dir and release tarballs at:

https://github.com/prometheus/statsd_exporter/releases

I am adding this to git annex, not sure if this is correct for
foreman-packaging.

This is the initial version for review, I haven't tested the package
yet. Once it passes build in Koji, I will download and test it
end-to-end on 1.18.

### CODE REVIEW

**Comment by lzap** [comment]
Amended changes.

File: packages/foreman/statsd_exporter/statsd_exporter-0.6.0.tar.gz [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../.git/annex/objects/Xj/pW/SHA256E-s1057602--f1dc722c9c3b43ed05b5ea7983091d2f6633a3d103550e1c3a876ab04fe646ec.tar.gz/SHA256E-s1057602--f1dc722c9c3b43ed05b5ea7983091d2f6633a3d103550e1c3a876ab04fe646ec.tar.gz
```
Comment (lzap) [inline packages/foreman/statsd_exporter/statsd_exporter-0.6.0.tar.gz]: This file comes from github, not sure if git annex handles that correctly.

**Comment by ekohl** [comment]
You put the package in plugins in `package_manifest.yaml` but all other locations are for the core repository. That's why you get the file doesn't exist error (though not very clearly).

**Comment by lzap** [comment]
I pushed a change, now I am using http://git-annex.branchable.com/tips/using_the_web_as_a_special_remote/ for the source tarball.

**Comment by lzap** [comment]
I rebased, but still testing. There was an issue with the user account. Do not merge, still working on this.

**Comment by ekohl** [comment]
Added waiting on contributor. Please do ping if you're ready for a review.

**Comment by lzap** [comment]
I am closing this, this does not give good results in production. Will file new PR.

**Pull request closed without merging**